xbmc/platform/android/activity/JNIXBMCConnectivityManagerNetworkCallback.cpp

```diff
@@ -0,0 +1,65 @@
+/*
+ *  Copyright (C) 2012-2023 Team Kodi
+ *  This file is part of Kodi - https://kodi.tv
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *  See LICENSES/README.md for more information.
+ */
+
+#include "JNIXBMCConnectivityManagerNetworkCallback.h"
+
+#include "CompileInfo.h"
+
+#include <androidjni/Context.h>
+#include <androidjni/jutils-details.hpp>
+
+using namespace jni;
+
+static std::string s_className =
```

Comment:
maybe use a `constexpr` instead of the static string?

Reply:
also why the `s_` prefix?

Reply:
`constexpr` cannot be used as `std::string` is not a literal. 

That's right, I remove the prefix.

Reply:
anonymous namespace would be ok.